tests/plugins/table/fakeselectionadvancetab.js

```diff
@@ -0,0 +1,28 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: dialogadvtab,table,tableselection */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+
+		// #579
+		'test advance table dialog for ignorig selection class': function() {
+			if ( !CKEDITOR.plugins.tableselection.isSupportedEnvironment ) {
+				assert.ignore();
+			}
+
+			var bot = this.editorBot;
+
+			// Add table with fake selection.
+			bot.setHtmlWithSelection( '<table border="1">[<tr><td>Cell1</td><td>Cell2</td></tr>]</table>' );
+
+			bot.dialog( 'tableProperties', function( dialog ) {
+				assert.areSame( '', dialog.getValueOf( 'advanced', 'advCSSClasses' ) );
+				dialog.hide();
```

Comment:
There is no need to hide dialog here, especially that it happens after assert so does not have any effect on the test result. This line can be simply removed.